README.md

```diff
@@ -39,7 +39,7 @@ Pull requests are welcome. I will appreciate any help on either enhancing existi
 * [Interface](https://github.com/sagar-jadhav/go-examples/blob/master/src/interfaces.go)
 * [Struct](https://github.com/sagar-jadhav/go-examples/blob/master/src/struct.go)
 * [Pointer](https://github.com/sagar-jadhav/go-examples/blob/master/src/pointer.go)
-
+* [Unit Testing](https://github.com/sagar-jadhav/go-examples/tree/master/src/unit_test)
```

Comment:
- Rename `unit_test` directory  to `test`
- Provide link to power.go in README with name -> `Math Functions`
- Instead of linking `unit_test` directory provide link to power_test.go with name -> `Unit Test`

Assign #5 to yourself

Thank you for contribution 👍

Reply:
Hi done with all the changes requested

Reply:
Thank you